src/modules.c

```diff
@@ -1,11 +1,18 @@
 #include "modules.h"
 
+//Return value of temperature sensor in degrees C
 void read_temperature()
 {
 
 }
 
-void read_lights()
+//Returns value of light sensor between 0 and 100
+float read_lights()
 {
-    
+	ADMUX = _BV(MUX0);
+	ADCSRA = _BV(ADSC);
+	loop_until_bit_is_clear(ADCSRA, ADSC);
+
+	float lightLevel = (float)ADCW / 1024 * 100;
```

Comment:
This line and the next one can be replaced with ``return (float) ADCW / 1024 * 100;``.